thorlcr/slave/slavmain.cpp

```diff
@@ -1788,6 +1788,7 @@ class CJobListener : public CSimpleInterface
                         StringAttr wuid, graphName;
                         StringBuffer soPath;
                         msg.read(wuid);
+                        saveWuidToFile(wuid);
```

Comment:
What happens if there are multiple slaves running in the same pod?  Should only 1 of them write the file?

Reply:
Right, yes, in a numWorkresPerPod>1 scenario, they all overwrite the same file, not good, not sure if in practice it would cause an issue..

I've opened a related JIRA : https://track.hpccsystems.com/browse/HPCC-31016 and issued a fix.

This JIRA/PR won't need to change.